octodns_ovh/__init__.py

```diff
@@ -41,6 +41,10 @@ class OvhProvider(BaseProvider):
         )
     )
 
+    @property
+    def SUPPORTS_ROOT_NS(self):
+        return True
```

Comment:
This can just be a property set up above with the other `SUPPORTS_*` bits since there's no logic involved it doesn't need to be a full fledged property accessor.

Reply:
Fixed